db/seeds.rb

```diff
@@ -1,3 +1,8 @@
+prior_logger     = Rails.logger
```

Comment:
@maestromac Instead of swapping out the logger, can't you just make a new one and then change the rest of this file to use it? 

```ruby
logger           = ActiveSupport::Logger.new(STDOUT)
logger.formatter = Rails.application.config.log_formatter
seeds_logger = ActiveSupport::TaggedLogging.new(logger)
```

And the use  `seeds_logger` in the rest of this file.

Reply:
Great suggestion! I did not think about thread-safety. I went ahead and applied this.